deploy/blobfuse2/config.yaml

```diff
@@ -0,0 +1,23 @@
+allow-other: true
+
+logging:
+  type: base
```

Comment:
Keeping default logging policy as "base" means logs will be dumped into a file, we shall rather go for "syslog" as default logging policy.

Reply:
I think that's the main reason I use this config.yaml for blobfuse2, let me try remove this config file and check whether blobfuse2 e2e test could pass: https://github.com/Azure/azure-storage-fuse/issues/734#issuecomment-1066061229